BigCommerce/Overview/01_General.md

```diff
@@ -0,0 +1,23 @@
+# General information
+
+*BigCommerce* is one of our partners in the MACH-Alliance. With its API-first approach to e-commerce, *BigCommerce* aims to simplify e-commerce on an enterprise level.   
+Through the integration of *BigCommerce* as an *Omni-Channel* extension, you will be able to connect your *BigCommerce* store(s) and thus manage all areas of work processes of online retailing centrally and from one place.
+
+The documentation refers to the *BigCommerce Integration* plugin version 1.1.0. There may be differences in the UI, features, and operation if your *BigCommerce Integration* plugin is installed in a different version.
+
+
+## Benefits
+
+The Actindo *BigCommerce Integration* plugin offers the following benefits:
+- Easy installation via the *Actindo Core1 DOP Connector* app in *BigCommerce*
+- Customizable connection settings for *BigCommerce* connection
+- Reliable data exchange between *Actindo* and *BigCommerce*.
+
+
+## Prerequisites
+
+- The *Actindo DataHub* module version 4.5.0 or higher is installed.
+- The *Omni-Channel* module version 4.2.1 or higher is installed.
+- The *Global Driver Management* module version 4.0.2 or higher is installed.<!---braucht man das??-->
+- The *UCS product sync* plugin version 4.3.0 or higher is installed. <!---für den Datentyp related products nötig?-->
+- The *BigCommerce Integration* plugin version 1.0.1 or higher is installed.
```

Comment:
BigCommerce Integration braucht man hier nicht anzugeben...

Die notwendigen Module/Plugins kannst du in Einstellungen > Platform Manager > Select BigCommerce Integration in der Liste installierten Module > ABHÄNGIGKEITEN ANZEIGEN

![image](https://github.com/Actindo-AG/Documentation/assets/99268392/f10203ab-514f-4f75-b1ad-3feb33b526f2)